autogluon/utils/tabular/ml/models/catboost/catboost_model.py

```diff
@@ -146,6 +146,11 @@ def _fit(self, X_train, y_train, X_val=None, y_val=None, time_limit=None, **kwar
 
         params = self.params.copy()
         num_features = len(self.features)
+        if num_gpus != 0:
```

Comment:
add comment that only 1 GPU can be utilized?

Reply:
I believe CatBoost can use multi-gpu.